app/models/validations/financial_validations.rb

```diff
@@ -200,7 +205,9 @@ def validate_rent_range(record)
         record.errors.add :rent_type, I18n.t("validations.financial.brent.rent_type.below_hard_min")
         record.errors.add :needstype, I18n.t("validations.financial.brent.needstype.below_hard_min")
         record.errors.add :period, I18n.t("validations.financial.brent.period.below_hard_min")
-      elsif record.beds.blank? || record.beds < LaRentRange::MAX_BEDS
+      end
+
+      if record.weekly_value(record["brent"]) > rent_range.hard_max
```

Comment:
this looks sensible

Reply:
Is it possible for there to be a `below_hard_min` and `above_hard_max` at the same time? That seems unlikely.